pkg/sql/catalog/tabledesc/structured.go

```diff
@@ -1252,387 +1254,6 @@ func (desc *Mutable) OriginalVersion() descpb.DescriptorVersion {
 	return desc.ClusterVersion.Version
 }
 
-type testingDescriptorValidation bool
-
-// PerformTestingDescriptorValidation can be set as a value on a context to
-// ensure testing specific descriptor validation happens.
-var PerformTestingDescriptorValidation testingDescriptorValidation = true
-
-// Validate performs ValidateSelf and then validates that
-// each reference to another table is resolvable and that the necessary back
-// references exist.
-func (desc *wrapper) Validate(ctx context.Context, descGetter catalog.DescGetter) error {
```

Comment:
Note: I moved this and the other validation functions to a new source file `validate.go`. The diffs were pretty gnarly anyway and as it turns out this also helps slim down `structured.go` which is far too big a source file.